### PR TITLE
Homework 2 - fizzbuzz and temperature converter

### PR DESCRIPTION
**Pull request by DanPitrowiski**
Also there were some slight updates to past homework and files

### CODE REVIEW

**Comment by DanPitrowiski** [comment]
Homework 3 has been added too.

**Comment by DanPitrowiski** [comment]
Changed the `index.html` file to reference homework 1 for my homework :dancer:

**Review by Borgaard — CHANGES_REQUESTED** [review]
Let's get your .DS_Store file situation sorted. I can't accept this pull request because that file is creating a merge conflict. 

.DS_Store is a hidden metadata file containing information about your homework folder, which we don't want because it's different for everyone and causes merge conflicts like this one.

We can do it before or after class, but I'd like to get it done before accepting your pull request. If you'd like to do it on your own, here's what you need to do:

- navigate to your homework directory
- `$ ls -la` to see your hidden files - they all start with a dot `.`.
-`$ git rm --cached .DS_Store` to delete the .DS_Store file in your root homework directory (```~/JSD/JS-SF-8-homework/.DS_Store```)
- `touch .gitignore` to create a special hidden file to prevent git from tracking certain files.
- open .gitignore and add the following 3 lines:
```
.DS_Store
*.DS_Store
.DS_Store?
```

When you've done that, git add, commit, and push. This will automatically update your pull request and you don't have to do anything more. You're all set! 😋 

Optional bonus, to always ignore .DS_Store files everywhere:
`$ echo .DS_Store >> ~/.gitignore_global` this creates a global ignore file and writes .DS_Store to it.
`git config --global core.excludesfile ~/.gitignore_global` to let git you want to use this file for all of your repositories.

File: danpitrowiski/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# whiski.github.io
```
Comment (Borgaard) [inline danpitrowiski/README.md:1]: Leads to a 404 :P 
I was expecting something else with a url like that!

**Comment by DanPitrowiski** [comment]
@Borgaard ok - changes have been made. I just removed `.DS_Store` all together. Should be good now :+1:

File: danpitrowiski/hw1-madlibs/index.html [hunk 2]
```diff
@@ -35,7 +35,7 @@ <h2 id="favorites"></h2>
 
   </h2>
 
-  <script src='http://code.jquery.com/jquery-latest.min.js'></script>
+  <script src='https://code.jquery.com/jquery-latest.min.js'></script>
```
Comment (Borgaard) [inline danpitrowiski/hw1-madlibs/index.html:38]: Nice catch 😜

**Pull request merged**